src/lowlevel/mpc.jl

```diff
@@ -0,0 +1,315 @@
+using RigidBodyDynamics: lower, upper, velocity_to_configuration_derivative_jacobian, velocity_to_configuration_derivative_jacobian!
+using Compat: adjoint
+
+struct MPCStage{C}
+    Δt::Float64
+    q::Vector{Variable}
+    v::Vector{Variable}
+    v̇::Vector{Variable}
+    u::Vector{Variable}
+    contacts::Vector{C}
+    initialized::typeof(Ref(false))
```

Comment:
On latest Julia master,

```julia
julia> typeof(Ref(false))
Base.RefValue{Bool}
```

So it doesn't seem like this has actually changed. It's fine the way it is, but just FYI.

Reply:
Oh, I thought `Ref` was becoming a concrete type in 0.7. OK, I'll make it more explicit